_config.yml

```diff
@@ -28,5 +28,3 @@ google_analytics_ua: UA-48605964-19
 
 sass:
   sass_dir: assets/_scss
-  style: :compressed
```

Comment:
I added an issue here: https://github.com/18F/govt-wide-patternlibrary/issues/28 let's go ahead and put it back in for now and address this in that issue. :+1: